h/static/scripts/bridge.coffee

```diff
@@ -118,7 +107,12 @@ class Bridge
       options = $.extend {}, options, {origin: '*'}
     channel = Channel.build(options)
 
+
+
+# Set up the bridge plugin, which bridges the main annotation methods
+# between the host page and the panel widget.
```

Comment:
This is not really accurate. The bridge plugin sets up a channel between frames and provides an events API on top of it. It does not care about the purpose of the messages.